test/TEST-16-ISSUE-518/avocado.conf

```diff
@@ -0,0 +1,2 @@
+[datadir.paths]
+logs_dir = /tmp/job-results
```

Comment:
Why do we need `avocado-framework`?

This test doesn't run tests inside `VM/container` (it uses the *installed* systemd, not the one from the built source tree). Why does this test in `test/TEST-*-ISSUE-*`?